pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -236,6 +243,11 @@ rec {
           ];
           __propagatedImpureHostDeps = computedPropagatedImpureHostDeps ++ __propagatedImpureHostDeps;
         };
+      # TODO: use dependencies/propagatedDependencies instead of just nativeBuildInputs?
```

Comment:
@Ericson2314, any help you could provide here as to the correct incantation/set of dependencies to use here would be appreciated. I used `nativeBuildInputs` to prototype this as `cmake` is always in `nativeBuildInputs`, at least AFAIK.